closed/OpenJ9.gmk

```diff
@@ -328,7 +328,6 @@ stage-j9 : stage-openj9-tools
 OPENJ9_VERSION_VARS := \
 	COMPILER_VERSION_STRING \
 	HOTSPOT_TARGET_OS \
-	OPENJ9_TAG \
```

Comment:
This is needed for `#define OPENJ9_TAG                "@OPENJ9_TAG@"` when processing closed/openj9_version_info.h.in

Reply:
I've reworked the solution without needing to change `OPENJ9_VERSION_VARS`.